vehicle/renault.go

```diff
@@ -270,15 +276,15 @@ func (v *Renault) jwtToken(sessionCookie string) (string, error) {
 	return res.IDToken, err
 }
 
-func (v *Renault) kamereonRequest(uri string) (kamereonResponse, error) {
-	data := url.Values{"country": []string{"DE"}}
+func (v *Renault) kamereonRequest(uri string, body io.Reader) (kamereonResponse, error) {
+	params := url.Values{"country": []string{"DE"}}
 	headers := map[string]string{
 		"x-gigya-id_token": v.gigyaJwtToken,
 		"apikey":           "Ae9FDWugRxZQAGm3Sxgk7uJn6Q4CGEA2", // v.kamereon.APIKey
```

Comment:
Api Key changed. New one seems to be VAX7XYKGfa92yMvXculCkEFyfZbuM7Ss (see https://github.com/jamesremuscat/pyze/issues/109#issuecomment-1073218876)